doc/user/content/sql/system-catalog/mz_internal.md

```diff
@@ -131,6 +131,17 @@ the most recent status for each AWS PrivateLink connection in the system.
 | `last_status_change_at` | [`timestamp with time zone`] | Wall-clock timestamp of the connection status change.|
 | `status` | [`text`] | | The status of the connection: one of `pending-service-discovery`, `creating-endpoint`, `recreating-endpoint`, `updating-endpoint`, `available`, `deleted`, `deleting`, `expired`, `failed`, `pending`, `pending-acceptance`, `rejected`, or `unknown`. |
 
+## `mz_cluster_blue_green_lineage`
+
+The `mz_cluster_blue_green_lineage` table shows the blue/green deployment lineage of all clusters in [`mz_clusters`](../mz_catalog/#mz_clusters). It determines all cluster IDs that are logically the same cluster.
```

Comment:
I'm finding this name/description to be a little confusing. Does it only contain data on clusters that actively undergoing a blue/green deployment, or does it contain historical data about clusters that have gone through blue/green deployments in the past?

Reply:
> Does it only contain data on clusters that actively undergoing a blue/green deployment, or does it contain historical data about clusters that have gone through blue/green deployments in the past?

Ah it contains historical data about clusters that have gone through blue/green deployments in the past, as well as current clusters! Brings a good point that I should probably make a testdrive/SLT test for this.

Reply:
Created a test in `test/sqllogictest/cluster_blue_green_lineage.slt`. Hopefully it better describes what I'm aiming for! I wonder if it makes sense to change the field description of `cluster_name` to "The ID of the cluster, regardless of whether it has been dropped or not. Corresponds to [`mz_clusters.id`](../mz_catalog/#mz_clusters)." ?